pom.xml

```diff
@@ -224,7 +224,7 @@
     <module>activemq-run</module>
     <module>activemq-shiro</module>
     <module>activemq-spring</module>
-    <module>activemq-runtime-config</module>
+    <!-- <module>activemq-runtime-config</module> -->
```

Comment:
Presumably a build break waiting to happen, only still working now as it can download a [always-stale] snapshot, but once you try to release it wont be able to. Essentially the same as the issues that cropped up during the 5.17.0 release.

(That or its entirely unused and should be properly removed rather than just commented out)